docs/TVM_EP.md

```diff
@@ -14,15 +16,17 @@
 ## Introduction
 
 TVM is an execution provider for ONNX Runtime that is built on top of Apache TVM. It enables ONNX Runtime users to leverage Apache TVM model optimizations.
-TVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux, but not on Windows or MacOS.
+TVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux or Windows, but not on MacOS.
 
 ## Build ONNX Runtime with the TVM Execution Provider
 
+### **Linux**
 Install the minimal pre-requisites on Ubuntu/Debian like linux operating systems:
 ```bash
 apt-get install -y python3 python3-dev python3-pip python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev llvm-12
-pip3 install numpy decorator attrs
+pip3 install numpy decorator attrs nasm
 ```
+Note: since ONNX Runtime with TVM EP is built with Intel ipp-crypto library there are new requirements. Compiler gcc (and g++) version should be equal to or higher than 8.2. nasm version should be 2.14.02 or higher. Problem with small nasm version can be seen [here](https://github.com/intel/ipp-crypto/issues/9) or [here](https://bugzilla.nasm.us/show_bug.cgi?id=3392205). For ubuntu LTS 18 `apt-get install nasm` is not enough due to it has version 2.13.02, see how to install from sources instruction [here](https://stackoverflow.com/questions/36144930/steps-to-install-nasm-offline-on-ubuntu#:~:text=wget%20http%3A//www.nasm.us/pub/nasm/releasebuilds/2.14.02/nasm%2D2.14.02.tar.bz2%0Atar%20xfj%20nasm%2D2.14.02.tar.bz2%0Acd%20nasm%2D2.14.02/%0A./autogen.sh%0A./configure%20%2D%2Dprefix%3D/usr/local/%20%0Amake%20%0Asudo%20make%20install%0Ahash%20%2Dd%20nasm).
```

Comment:
I would copy paaste the title of the page instead of here. It seems better for the user.

Reply:
Good point, done